### ISSUE TITLE
Amazfit Bip: Connection issue with firmware 0.1.0.66

### ISSUE THREAD

**Issue by kantrol** [Freeyourgadget/Gadgetbridge#969]
#### Your issue is:
I could not connect to Amazfit Bip after update to newest firmware 0.1.0.66. I finally recognized that the update **changed the Bluetooth MAC Address** of the watch. I had to add a new device in gadgetbridge.

Never saw any device changing MAC address on firmware update before!!

The only problem now is that I need to merge the history of what has been stored in gadgetbridge with the previous MAC address to the current device.

The original issue (MAC address) is not an issue of gadgetbridge, but the consquences are.

#### Your wearable device is:
Amazfit Bip Firmware 0.1.0.66

#### Your android version is:
7.1
#### Your Gadgetbridge version is:

0.24.1

**Comment by ashimokawa** [comment]
@kantrol:
This did not happen to me when updating to 0.1.0.66.
It does happen when factory resetting the watch! Did you do that?

**Comment by ribee** [comment]
i have the same issue on 0.1.0.86.
I connected the bip for the first time and it did not work correctly. Since then I have an entry in Gadgetbridge that I can not delete and I see the Bip in my Bluetooth devices in android and can not disconnect the connection. Then I updated to the beta 0.1.0.89 and I was able to connect properly. But I now have two Bip in Gadgetbrigde with different Mac addresses

**Comment by LHolst** [comment]
similar issue here,
i update mi band 3 with miFit app and now the band shows a changed MAC.
now i have a new device entry next to the old one.

old one has the data, i like to merge these

**Comment by wedme** [comment]
Yep, did exactly the same thing with an Amazfit Bip. Did a factory reset and MAC changed. Now gadgetbridge added it as a new device and all my data is on the old one. Is there a way to manually change the MAC on the Bip? If not then at least change the reference MAC on all the Database data to match the new MAC before import? I don't mind doing it manually at file level if someone knows which files. Thanks ;)
